packages/Ludown/test/ludown.negative.test.suite.js

```diff
@@ -51,7 +51,7 @@ describe('Negative tests', function() {
     it('should show WARN message when an answer is missing for qna pair', function(done) {
         exec(`node ${ludown} parse toqna --in ${TEST_ROOT}/testcases/bad.lu`, (error, stdout, stderr) => {
             try {
-                assert.equal(stderr.includes('No answer found for question:'), true);    
+                assert.equal(stderr.includes("syntax error: mismatched input '$' expecting {'**Filters:**', MULTI_LINE_TEXT}"), true);    
```

Comment:
Same here. We should improve the error message so it is more actionable. In all of these, can we include line/ character position or do we not have that information available at this level?

Reply:
Actually the error message does have line/column information in it(I didn't include them in the assert since it is using stderr.includes()). I have updated the tests here to add the line/column information in the assert string.

Reply:
The actual error message is "**[ERROR] line 3:0 - line 3:1: syntax error: mismatched input '$' expecting {'**Filters:**', MULTI_LINE_TEXT}**"